tests/e2e-playwright/tests/conftest.py

```diff
@@ -323,6 +323,11 @@ def _do() -> AutoRegisteredUser:
     return _do
 
 
+@pytest.fixture(scope="session")
+def store_browser_context() -> bool:
+    return False
+
```

Comment:
why you don't use a pytest_option for this? it could just default to False that way.
see `make test-sleepers-anywhere` or similar on how to create the configuration that can be used for testing.,

Actually my goal at some point was to remove `make test-sleepers` abd `make test-sleepers-dev` to replace with the above.